pkg/util/interrupt/child_linux.go

```diff
@@ -24,4 +24,4 @@ import (
 )
 
 // childSignals are the allowed signals that can be sent to children in Unix variant OS's
-var childSignals = []os.Signal{syscall.SIGINT, syscall.SIGTERM, syscall.SIGQUIT}
```

Comment:
Currently, the distinction we're making is not Windows x Others, but Linux x Others.